jdaviz/configs/imviz/helper.py

```diff
@@ -22,22 +15,19 @@
 
 __all__ = ['Imviz']
 
-ASTROPY_LT_4_3 = not minversion('astropy', '4.3')
-RESERVED_MARKER_SET_NAMES = ['all']
-
 
 class Imviz(ConfigHelper):
-    """Imviz Helper class"""
+    """Imviz Helper class."""
     _default_configuration = 'imviz'
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self._default_viewer = self.app.get_viewer('imviz-0')
 
-        # Markers
-        self._marktags = set()
-        self._default_mark_tag_name = 'default-marker-name'
-        # marker shape not settable: https://github.com/glue-viz/glue/issues/2202
-        self.marker = {'color': 'red', 'alpha': 1.0, 'markersize': 5}
+    @property
+    def default_viewer(self):
+        """Default viewer instance."""
```

Comment:
```suggestion
        """Default viewer instance. This is typically the first viewer ("imviz-0")."""
```
I used the word "typically" here as a weasel word in case we add machinery to change it, but I think it's good to say this since it's probably 95% of the time that this is true (i.e., the user would have to rename the user explicitly to make it not true I think?)